imap_processing/cdf/config/imap_idex_l1_variable_attrs.yaml

```diff
@@ -0,0 +1,420 @@
+int_fillval: &int_fillval -9223372036854775808
+min_epoch: &min_epoch -315575942816000000
+max_epoch: &max_epoch 3155630469184000000
+
+data_min: &data_min 0
+data_max: &data_max 4096
+
+sample_rate_min: &sample_rate_min -130
+sample_rate_max: &sample_rate_max 130
+
+int_maxval: &int_maxval 9223372036854775807 # np.iinfo(np.int64).max
```

Comment:
I hadn't realized you can define commonly-used values like this. I am going to steal this for CoDICE!